### ISSUE TITLE
Added device to user account but device does not display "my Devices" page for that user

### ISSUE THREAD

**Issue by phpmaven** [Ylianst/MeshCentral#3931]
Created a new user
Gave that user access to one device
User logs in
My Devices page lists no available devices

Interestingly, if I add the device group that the device is to the user account, those devices all show up.
If I remove that device group from the account, then it leaves the one device under "Individual Devices" until I refresh the page, then it disappears.

Thanks,

Mark

**Comment by phpmaven** [comment]
If it helps, this is using MariaDB ver 10.2.43 on Centos 7

**Comment by githubbiswb** [comment]
Can confirm this but, not with mariadb, using the following docker image build which auto builds from this repo on alipine linux. 

My server version is 1.0.31

https://github.com/gurucomputing/meshcentral-docker

**Comment by githubbiswb** [comment]
I am now on 1.0.38 and also wanted to add it displays just fine for me in the GUI but not in the Mesh Router interface

**Comment by githubbiswb** [comment]
@si458 and @Ylianst I think this can be closed as the fix with 4188 would take care of this issue as well

**Comment by si458** [comment]
yes it can be closed as @Ylianst released a new `meshcentralrouter.exe` update to fix the issue!, ive no permission to close sadly
